src/main/java/com/health/model/QueueManagement.java

```diff
@@ -0,0 +1,510 @@
+        return endTime;
+    }
+
+    public void setEndTime(long endTime) {
+        this.endTime = endTime;
+    }
+
+    public long getProcesingTime() {
+        return procesingTime;
+    }
+
+    public void setProcesingTime(long procesingTime) {
+        this.procesingTime = procesingTime;
+    }
+
+    public QueueManagement() {
+        super();
+
+    }
+
+    public QueueManagement(long queueId, Timestamp request, String requestType, String status, long startTime,
+            long endTime, long procesingTime) {
+        super();
+        this.queueId = queueId;
+        this.requestTime = request;
+        this.requestType = requestType;
+        this.status = status;
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.procesingTime = procesingTime;
+
+    }
+
+    public QueueManagement(long queueId, Timestamp requestTime, String requestType) {
+        super();
+        this.queueId = queueId;
+        this.requestTime = requestTime;
+        this.requestType = requestType;
+    }
+
+    public QueueManagement(long queueId, String documentId, String documentType, int rank, String language,
+            int languageId, String status) {
+        super();
+        this.queueId = queueId;
+        this.documentId = documentId;
+        this.documentType = documentType;
+        this.rank = rank;
+        this.language = language;
+        this.languageId = languageId;
+        this.status = status;
+    }
+
+    @Override
+    public String toString() {
+        return "QueueManagement [queueId=" + queueId + ", requestTime=" + requestTime + ", requestType=" + requestType
+                + ", status=" + status + ", documentId=" + documentId + ", documentType=" + documentType + ", rank="
+                + rank + ", language=" + language + "]";
+    }
+
+    @Override
+    public void run() {
+
+        StringBuilder documentSb = new StringBuilder();
+        String api_url = "";
+
+        setStartTime(System.currentTimeMillis());
+        setStatus(CommonData.STATUS_PROCESSING);
+
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            HttpRequest request = null;
+            HttpPost httpPost = null;
+            HttpGet httpGet = null;
+            HttpResponse response = null;
+
+            if (getRequestType().equals(CommonData.ADD_DOCUMENT)
+                    || getRequestType().equals(CommonData.UPDATE_DOCUMENT)) {
+
+                if (getRequestType().equals(CommonData.ADD_DOCUMENT)) {
+                    documentSb.append(commonData.elasticSearch_url);
+                    documentSb.append("/");
+                    documentSb.append("addDocument");
+                    documentSb.append("/");
+                    documentSb.append(documentId);
+                    documentSb.append("/");
+                    documentSb.append(documentType);
+                    documentSb.append("/");
+                    documentSb.append(languageId);
```

Comment:
These look like common lines (please confirm).

In that case, put these lines before the first if. Replace "addDocument" with getRequestType().